layers/meta-resin-genericx86/recipes-support/resin-init/resin-init-flasher.bbappend

```diff
@@ -1,6 +1,6 @@
 # On Intel genericx86-64, we flash on internal hard disk drive
 # The first one found will be used
-INTERNAL_DEVICE_KERNEL = "nvme0n1 sda"
+INTERNAL_DEVICE_KERNEL = "nvme0n1 sda mmcblk0"
```

Comment:
@telphan can we also add "sdb" to this list here?
Apparently, there may be cases where "sda" is actually the USB stick. In such cases, commit 7a39577e0a36427e82efeab9a57b609f2c0e7c38 will help making resin-init-flasher not flashing the USB stick from which it is running, but at the same time it won't be able to flash the internal media (in the case when the USB stick is detected as sda, the internal media will be sdb)